test/fixtures/@typescript-eslint/no-dynamic-delete.pass.ts

```diff
@@ -1,9 +1,8 @@
-/* eslint-disable @ridedott/no-single-line-comment-block */
 /* eslint-disable @typescript-eslint/no-magic-numbers */
 /* eslint-disable functional/immutable-data */
 
 const containerPass: { [i: string]: number } = {
-  /* ... */
```

Comment:
This one conflicted with Prettier: ESLint wanted an empty line before the comment, Prettier removed it. Since out @ridedott/eslint-plugin doesn't like single-line comments blocked anyway, I updated this one and it solved it.